tgui/packages/tgui/interfaces/ResearchMemories.js

```diff
@@ -0,0 +1,108 @@
+import { useBackend, useLocalState } from '../backend';
+import {
+  Button,
+  Flex,
+  Section,
+  Box,
+  Tabs,
+  Fragment,
+  LabeledList,
+} from '../components';
+import { Window } from '../layouts';
+
+export const ResearchMemories = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [clueCategory, setClueCategory] = useLocalState(
+    context,
+    'clueCategory',
+    0
+  );
+
+  const { clearance, research_credits, theme, clue_categories } = data;
+
+  return (
+    <Window width={650} height={700} theme={theme}>
+      <Window.Content scrollable>
+        <Section title={'Clearance: ' + clearance}>
+        </Section>
+        <Section title={'Research Credits: ' + research_credits}>
+        </Section>
```

Comment:
empty fragments autoclose, linter fail. you might wanna use something other than sections for this though

Reply:
What would you suggest? I'm really bad at JS...

Reply:
Changed it to Flex.Item